client/src/timeline.js

```diff
@@ -404,24 +409,28 @@ Chart.defaults.timeline = {
             categoryPercentage: 0.8,
             barPercentage: 0.9,
             offset: true,
+          //offset: false,
```

Comment:
Can we just remove lines instead of commenting them out? Same for lines 414 and 415 and 432